README.md

```diff
@@ -12,59 +12,57 @@
-| 7      |  Differential privacy, Ethics, Inequity  | | [Slides 1](https://github.com/UofT-DSI/sampling/blob/main/lectures/14-Differential%20privacy-slides.pptx) <br> [Slides 2](https://github.com/UofT-DSI/sampling/blob/main/lectures/15-Ethics-slides.pptx) <br> [Slides 3](https://github.com/UofT-DSI/sampling/blob/main/lectures/16-Inequity-slides.pptx) |  
-| 8      |    Case Study                               | [Final Assessment](https://github.com/UofT-DSI/sampling/blob/main/assessment/Final%20Assessment.md) | |
+-	Monday March 25 6:00-8:30 PM (Course Overview, Sampling Basics)
+-	Tuesday March 26 6:00-8:30 PM (Probability 101)
+-	Wednesday March 27 6:00-8:30 PM (Populations and Samples)
+-	Thursday March 28 6:00-8:30 PM (Types of Sampling)
+-	Tuesday April 2 6:00-8:30 PM (Types of Sampling (continued))
+-	Wednesday April 3 6:00-8:30 PM (Errors in Sampling)
+-	Thursday April 4 6:00-8:30 PM (Survey Quality and Questionnaire Design)
+-	Saturday April 6 9:00-11:30 AM (Sampling Ethics)
+-	Monday April 8 ASYNCHRONOUS CLASS (Survey Quality Continued))
+-	Tuesday April 9 6:00-8:30 PM (Sampling Practice)
+-	Thursday April 11 6:00-8:30 PM (Guest Speaker)
 
 
 ## Marking Scheme and Assignment Submission
-Please submit your assignments as pdf files using the google forms linked in the table below.
-| Assessment       | Weight |  Due Date | Submission Link |
-|------------------|--------|-----------|-----------------|
-| Participation |  10%  |        |
-| [Questionnaire Design (Part A)](https://github.com/UofT-DSI/sampling/blob/main/assessment/ASSIGNMENT%20-%20Questionnaire%20Design%20(Part%20A).md) |  10%  |   March 5, 2024      | [https://forms.gle/pQW2mwzSXPSXcchB9](https://forms.gle/pQW2mwzSXPSXcchB9) |
-| [Questionnaire Design (Part B)](https://github.com/UofT-DSI/sampling/blob/main/assessment/ASSIGNMENT%20-%20Questionnaire%20Design%20(Part%20B).md) |  10% | March 5, 2024 | [https://forms.gle/xHSfCMq8QAa2PUyQA](https://forms.gle/xHSfCMq8QAa2PUyQA) |
-| [Data Documentation Comparison Worksheet](https://github.com/UofT-DSI/sampling/blob/main/assessment/ASSIGNMENT%20-%20Data%20Documentation%20Comparison%20Worksheet.md) |  15% |  March 12, 2024  | [https://forms.gle/voo6ndjo7wjY72Zv8](https://forms.gle/voo6ndjo7wjY72Zv8)|
-| [Sampling and Reproducibility in Python](https://github.com/UofT-DSI/sampling/blob/main/assessment/ASSIGNMENT%20-%20Sampling%20and%20Reproducibility.md) |   15%  |  March 12, 2024        | [https://forms.gle/SwsuDdrxiwNJVWdd6](https://forms.gle/SwsuDdrxiwNJVWdd6)|
-| [Final Assessment](https://github.com/UofT-DSI/sampling/blob/main/assessment/Final%20Assessment.md) |   40%  |  March 9, 2024   |[https://forms.gle/QDbissy89WnY3JQP8](https://forms.gle/QDbissy89WnY3JQP8)|
+Rubrics for each assignment (pass/fail) are included in each assignment document in the 'assessments' folder. Please submit your assignments using the google forms linked in the table below.
+-	Assignment 1: https://forms.gle/GoAkM7f8vKCFaVow5 
```

Comment:
this is not necessary as we are using a google drive mechanism to intake their submissions.